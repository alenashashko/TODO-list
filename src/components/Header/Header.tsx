import { FC } from "react";
import { AppBar, Toolbar, Typography } from "@material-ui/core";

import { useStyles } from "./Header.styles";

export const Header: FC = () => {
  const classes = useStyles();

  return (
    <AppBar position="static">
      <Toolbar>
        <Typography className={classes.title} variant="h6" component="h1">
          Todo List
        </Typography>
      </Toolbar>
    </AppBar>
  );
};
