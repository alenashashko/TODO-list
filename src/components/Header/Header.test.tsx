import { render, cleanup } from "@testing-library/react";
import { Header } from "./Header";

afterEach(cleanup);

it("renders header", () => {
  const { asFragment } = render(<Header />);

  expect(asFragment()).toMatchSnapshot();
});
