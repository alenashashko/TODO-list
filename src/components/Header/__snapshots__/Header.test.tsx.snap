// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h1
        class="MuiTypography-root makeStyles-title-1 MuiTypography-h6"
      >
        Todo List
      </h1>
    </div>
  </header>
</DocumentFragment>
`;
